landing_page/db/schema.rb

```diff
@@ -43,13 +43,23 @@
     t.index ["key"], name: "index_active_storage_blobs_on_key", unique: true
   end
 
+  create_table "archivedposts", force: :cascade do |t|
+    t.string "title"
+    t.text "body"
+    t.integer "user_id", null: false
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
+    t.index ["user_id"], name: "index_archivedposts_on_user_id"
+  end
+
```

Comment:
Why do you need this table?

Reply:
Resolved in commit 084d812ae08e7a35c71b40da5aca618a372678fa